src/org/parosproxy/paros/extension/option/OptionsParamView.java

```diff
@@ -91,6 +91,8 @@
 	public static final String LARGE_RESPONSE_SIZE = "view.largeResponse";
 	public static final String FONT_NAME = "view.fontName";
 	public static final String FONT_SIZE = "view.fontSize";
+	public static final String HTTP_MESSAGE_FONT_NAME = "view.httpMessageFontName";

```

Comment:
Same here and following changes.

Note that this is a Paros class, it should be added a ZAP comment after the license header:
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines